File: 1-Iniciante/JavaScript/1008.js
let numero_funcionario = parseInt(lines.shift());
let horas = parseInt(lines.shift());
let qtd_recebe_horas = parseFloat(lines.shift());

let salario = horas * qtd_recebe_horas;

console.log(`NUMBER = ${numero_funcionario}\nSALARY = U$ ${salario.toFixed(2)}`);

File: 1-Iniciante/Dart/1008.dart
import 'dart:io';

void main(){
  int number, horas;
  double valorHora;
    number = int.parse(stdin.readLineSync());
    horas = int.parse(stdin.readLineSync());
    valorHora = double.parse(stdin.readLineSync());
    var salario = (horas*valorHora).toStringAsFixed(2);
    
    print("NUMBER = $number");
    print("SALARY = U\$ $salario");    
}